app/containers/Homebrew.js

```diff
@@ -0,0 +1,176 @@
+                      'Mint',
+                      'Soda Water',
+                      'Guinness',
+                      'Bass Pale Ale',
+                      'Ginger Beer',
+                      'Goldschlager',
+                      'Chambord',
+                      'Drambuie',
+                      'Scotch',
+                      'Skittles',
+                      'Hard Cider',
+                      'Salt',
+                      'Rumpleminze',
+                      'Grand Marnier',
+                      'Pineapple',
+                      'Ginger Ale',
+                      'Jameson',
+                      'Banana',
+                      'Pina Colada Mix',
+                      'Raspberry Liqueur',
+                      'Grapefruit Juice',
+                      'Cointreau',
+                      'Tangelo Juice',
+                      'Lemon',
+                      'Alize Cognac',
+                      'Cinnamon',
+                      'Brandy',
+                      'Lime',
+                      'Cream of Coconut',
+                      'Cherry',
+                      'Chambord',
+                      'Lemonade',
+                      'Iced Tea',
+                      'Kool-Aid',
+                      'Benedictine',
+                      'Light Cream',
+                      ],
+    };
+  },
+
+  toggleIngredient: function (ingredient) {
+
+    return function () {
+
+      if (selectedIngredients.indexOf(ingredient) === -1) {
+        selectedIngredients.push(ingredient);
+        this.state.ingredientList.splice(this.state.ingredientList.indexOf(ingredient), 1);
+      } else {
+        this.state.ingredientList.push(ingredient);
+        selectedIngredients.splice(selectedIngredients.indexOf(ingredient), 1);
+      }
+
+      console.log(this.state.ingredientList.length);
+      console.log('Selected ingredients ' + selectedIngredients);
+
+    }.bind(this);
+
+  },
+
+  onSubmit: function () {
+    var finalDrinkList = [];
+    for (k = 0; k < this.state.drinkList.length; k++) {
+      finalDrinkList[k] = this.state.drinkList[k].ingredients;
+    }
+
+    console.log(finalDrinkList.length);
+    for (i = 0; i < this.state.ingredientList.length; i++) {
+      console.log('first loop');
+      console.log(this.state.ingredientList[i]);
+      for (j = 0; j < finalDrinkList.length; j++) {
+        //console.log(finalDrinkList[j]);
+        if (finalDrinkList[j].indexOf(this.state.ingredientList[i]) > -1) {
+
+          this.state.drinkList.splice(j, 1, '');
+        }
+      }
+    }
+
+    while (this.state.drinkList.indexOf('') > -1) {
+      this.state.drinkList.splice(this.state.drinkList.indexOf(''), 1);
+      console.log('snagged');
+    }
+
+    for (l = 0; l < this.state.drinkList.length; l++) {
+      console.log('Available drinks are : ');
+      console.log(this.state.drinkList[l].name + ' ' + this.state.drinkList[l].rating);
+
+    }
+
+  },
+
+  showIngredientList: function () {
```

Comment:
show implies it was built already, now your just showing it (like toggling the css property `visibility`). something like `renderIngredients` or `renderIngredientListItems`